tests/unittest/llmapi/apps/test_harmony_parsing.py

```
@@ -1327,6 +1327,109 @@ def test_stream_options_none_defaults_to_include(self):
+    adapter with a malformed header (no <|message|> token), the whole
+    message was dropped with no log output at all. The response then looked
+    like a normal refusal to call the tool -- HTTP 200, finish_reason
+    "stop", populated reasoning_content, and an empty tool_calls list --
+    which no caller can distinguish from the model declining to act.
+    """
+
+    @pytest.fixture
+    def adapter(self):
+        try:
+            return HarmonyAdapter(harmony_input=False, harmony_output=False)
+        except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)